src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -211,3 +212,6 @@ html {
 
 // Theming
 build-themes-separatedPanelPrefsEditorFrame(themes);
+
+// Fonts
+build-fonts-prefs(fonts);
```

Comment:
@jobara If we can remove those functions from those styl files, then we can probably remove them from Themes.styl too, right?